packages/framework/presence/tsconfig.json

```diff
@@ -7,5 +7,6 @@
 		"outDir": "./lib",
 		"noImplicitAny": true,
 		"noImplicitOverride": true,
+		"exactOptionalPropertyTypes": false,
```

Comment:
Somehow, the upstream import changes made in this PR caused this package to start failing due to many violations in _other_ upstream packages (e.g. `merge-tree`). I took a stab at cleaning up those violations, but there were _many_. I'm not sure how practical it would be to do.

Note that this PR does not strictly _need_ to be merged (at least not right now), but it would be nice to do.